tests/foreman/ui/test_org.py

```diff
@@ -878,28 +897,32 @@ def test_remove_environment(self, env_name):
         """
         strategy, value = common_locators['entity_select']
         strategy1, value1 = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        env = entities.Environment(name=env_name).create_json()
-        self.assertEqual(env['name'], env_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name, envs=[env_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_env'])
-            element = session.nav.wait_until_element((strategy1,
-                                                      value1 % env_name))
-            # Item is listed in 'Selected Items' list and not 'All Items' list.
-            self.assertIsNotNone(element)
-            self.org.update(org_name, envs=[env_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_env'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % env_name))
-            # Item is listed in 'All Items' list and not 'Selected Items' list.
-            self.assertIsNotNone(element)
+            for env_name in valid_env_names():
```

Comment:
Same comment as before about using one single organization and adding entity to it and asserting its presence. Then, remove entity and assert that it is gone from the organization. Not a blocker for this PR.